### PR TITLE
Remove CQL batching from connector

### PR DESCRIPTION
**Pull request by avelanarius**
Remove batching from `ScyllaDbSinkTask`. The rows are now inserted one-by-one in parallel (1 Future per 1 INSERT). 

Using (unlogged) batches was not correct as:
1. The heuristic calculating batch size was incorrect, resulting in warning log messages (batch too big).
2. The batches could span multiple Scylla partitions, in which case they are not recommended and could lead to degraded performance.

I observed better performance and better CPU utilization after this fix.

I also removed inserting offsets in (logged!) batches - I don't think there was any reason for it, and this logged batch could touch rows in many different Scylla nodes (across multiple partitions).

Removed `scylladb.timestamp.resolution.ms`, as this is not relevant if we are not doing batches. Similarly, removed `scylladb.max.batch.size.kb`.

### CODE REVIEW

File: src/main/java/io/connect/scylladb/ScyllaDbSession.java [hunk 1]
```diff
@@ -78,11 +78,13 @@
...
-     * @param batch statement batch that will be written to ScyllaDb
-     * @param offsetStates The list of SinkOffsetStates for the current batch of SinkRecords.
+     * Method generates a BoundStatement, that inserts the offset metadata
+     * for a given topic and partition.
+     * @param topicPartition topic and partition for the offset
+     * @param metadata offset metadata to be inserted
+     * @return statement that inserts the provided offset to Scylla.
      */
-    void addOffsetsToBatch(BatchStatement batch, Map<TopicPartition, OffsetAndMetadata> offsetStates);
+    BoundStatement getInsertOffsetStatement(TopicPartition topicPartition, OffsetAndMetadata metadata);
```
Comment (haaawk) [inline src/main/java/io/connect/scylladb/ScyllaDbSession.java:87]: I think this belongs to another commit, no?

**Review by haaawk — APPROVED** [review]
LGTM

File: src/main/java/io/connect/scylladb/ScyllaDbSession.java (hunk 1 above)
Comment (avelanarius) [reply src/main/java/io/connect/scylladb/ScyllaDbSession.java:87]: Yes, fixed.

**Comment by tarzanek** [comment]
couldn't this just batch by partition? then you would be able to get more out of it than single statements
(and I originally thought they batch by PK)

**Comment by avelanarius** [comment]
Previously it was not batching correctly:
1. It was not batching by PK - in a single batch there were multiple PKs
2. Batch size calculation was off - generated batches were too large, causing thousands of warnings in Scylla log ("batch too large")

Of course, I don't reject the possibility of bringing this back in the future (properly implemented). But to do it properly, we should do proper benchmarks and measure its speedup. In the meantime, I made a decision to remove this functionality.

The other consideration is: if we batched by PK, would we gain anything in a typical Kafka workload? By default, messages in Kafka are partitioned by hash of key. That means that the messages are partitioned by: `murmur2(avro/json(pk1, pk2, ..., ck1, ck2))` which is a different partitioning scheme than Scylla's `murmur3(pk1, pk2, ...)` - messages in the same Scylla partition are in different Kafka partitions.
